#!/bin/bash
#SBATCH -J ov4Rep14h
#SBATCH -p super
#SBATCH -N 1
#SBATCH -t 1-20:35:30
#SBATCH --output=ov4Rep14h_%j.out
#SBATCH --error=ov4Rep14h_%j.err
#SBATCH --mail-user=asna.amjad@utsouthwestern.edu
#SBATCH --mail-type=BEGIN
#SBATCH --mail-type=END
#SBATCH --mail-type=FAIL


#cd /project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/align-star-se.sh-1.0.0

#./compare-expression-cufflinks.sh -o 'OV3_All_Samples' -e 'DMSO2hr,DMSO4hr,BI2hr,BI4hr,BYK2hr,BYK4hr,BYKBI2hr,BYKBI4hr' -a 'OV3_DMSO_2_REP1_S1_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam,OV3_DMSO_2_REP2_S17_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_DMSO_4_REP1_S5_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam,OV3_DMSO_4_REP2_S21_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BI_2_REP1_S3_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam,OV3_BI_2_REP2_S19_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BI_4_REP1_S7_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam,OV3_BI_4_REP2_S23_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_2_REP1_S2_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam,OV3_BYK_2_REP2_S18_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_4_REP1_S6_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam,OV3_BYK_4_REP2_S22_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_plus_BI_2_REP1_S4_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam,OV3_BYK_plus_BI_2_REP2_S20_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_plus_BI_4_REP1_S8_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam,OV3_BYK_plus_BI_4_REP2_S24_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/merge_transcripts_OV3/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'

#./compare-expression-cufflinks.sh -o 'OV3_BI_2h_REP1' -e 'DMSO2hr,BI2hr' -a 'OV3_DMSO_2_REP1_S1_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BI_2_REP1_S3_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/merge_transcripts_OV3/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BI_2h_REP1_NEW' -e 'DMSO2hr,BI2hr' -a 'OV3_DMSO_2_REP1_S1_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BI_2_REP1_S3_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BI_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BYK_plus_BI_2h_REP1' -e 'DMSO2hr,BYKBI2hr' -a 'OV3_DMSO_2_REP1_S1_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_plus_BI_2_REP1_S4_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BYK_plus_BI_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BYK_plus_BI_4h_REP1' -e 'DMSO4hr,BYKBI4hr' -a 'OV3_DMSO_4_REP1_S5_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_plus_BI_4_REP1_S8_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BYK_plus_BI_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BI_4h_REP1' -e 'DMSO4hr,BI4hr' -a 'OV3_DMSO_4_REP1_S5_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BI_4_REP1_S7_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BI_4h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BYK_4h_REP1' -e 'DMSO4hr,BYK4hr' -a 'OV3_DMSO_4_REP1_S5_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_4_REP1_S6_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BYK_4h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'


## CuffDiff OV3 REP2
#./compare-expression-cufflinks.sh -o 'BI_2h_REP2' -e 'DMSO2hr,BI2hr' -a 'OV3_DMSO_2_REP2_S17_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BI_2_REP2_S19_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/merge_transcripts_OV3/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BYK_2h_REP2' -e 'DMSO2hr,BYK2hr' -a 'OV3_DMSO_2_REP2_S17_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_2_REP2_S18_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BI_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BYK_plus_BI_2h_REP2' -e 'DMSO2hr,BYKBI2hr' -a 'OV3_DMSO_2_REP2_S17_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_plus_BI_2_REP2_S20_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BYK_plus_BI_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'

#./compare-expression-cufflinks.sh -o 'BI_4h_REP2' -e 'DMSO4hr,BI4hr' -a 'OV3_DMSO_4_REP1_S21_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BI_4_REP2_S23_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BI_4h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BYK_4h_REP2' -e 'DMSO4hr,BYK4hr' -a 'OV3_DMSO_4_REP1_S21_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_4_REP2_S22_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BYK_4h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'BYK_plus_BI_4h_REP2' -e 'DMSO4hr,BYKBI4hr' -a 'OV3_DMSO_4_REP1_S21_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_plus_BI_4_REP2_S24_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/BYK_plus_BI_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'

## CuffDiff OV4 REP1
./compare-expression-cufflinks.sh -o 'ov4_BI_2h_REP1' -e 'DMSO2hr,BI2hr' -a 'OV4_DMSO_2_REP1_S9_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BI_2_REP1_S11_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BI_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
./compare-expression-cufflinks.sh -o 'ov4_BYK_2h_REP1' -e 'DMSO2hr,BYK2hr' -a 'OV4_DMSO_2_REP1_S9_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BYK_2_REP1_S10_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BYK_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
./compare-expression-cufflinks.sh -o 'ov4_BYK_plus_BI_2h_REP1' -e 'DMSO2hr,BYKBI2hr' -a 'OV4_DMSO_2_REP1_S9_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BYK_plus_BI_2_REP1_S12_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BYK_plus_BI_2h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'

./compare-expression-cufflinks.sh -o 'ov4_BI_4h_REP1' -e 'DMSO4hr,BI4hr' -a 'OV4_DMSO_4_REP1_S13_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BI_4_REP1_S15_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BI_4h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
./compare-expression-cufflinks.sh -o 'ov4_BYK_4h_REP1' -e 'DMSO4hr,BYK4hr' -a 'OV4_DMSO_4_REP1_S13_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BYK_4_REP1_S14_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BYK_4h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
./compare-expression-cufflinks.sh -o 'ov4_BYK_plus_BI_4h_REP1' -e 'DMSO4hr,BYKBI4hr' -a 'OV4_DMSO_4_REP1_S13_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BYK_plus_BI_4_REP1_S16_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BYK_plus_BI_4h_REP1_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'


## CuffDiff OV4 REP2
#./compare-expression-cufflinks.sh -o 'ov4_BI_2h_REP2' -e 'DMSO2hr,BI2hr' -a 'OV4_DMSO_2_REP2_S25_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BI_2_REP2_S19_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BI_2h_REP2_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'ov4_BYK_2h_REP2' -e 'DMSO2hr,BYK2hr' -a 'OV4_DMSO_2_REP2_S25_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_2_REP2_S18_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BYK_2h_REP2_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'ov4_BYK_plus_BI_2h_REP2' -e 'DMSO2hr,BYKBI2hr' -a 'OV4_DMSO_2_REP2_S25_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV3_BYK_plus_BI_2_REP2_S20_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BYK_plus_BI_2h_REP2_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'

#./compare-expression-cufflinks.sh -o 'ov4_BI_4h_REP2' -e 'DMSO4hr,BI4hr' -a 'OV4_DMSO_4_REP2_S29_R1_001_S21_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BI_4_REP2_S31_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BI_4h_REP2_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'ov4_BYK_4h_REP2' -e 'DMSO4hr,BYK4hr' -a 'OV4_DMSO_4_REP2_S29_R1_001_S21_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BYK_4_REP2_S30_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BYK_4h_REP2_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
#./compare-expression-cufflinks.sh -o 'ov4_BYK_plus_BI_4h_REP2' -e 'DMSO4hr,BYKBI4hr' -a 'OV4_DMSO_4_REP2_S29_R1_001_S21_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam OV4_BYK_plus_BI_4_REP2_S32_R1_001.fastq.gz.Aligned.sortedByCoord.out.sorted.bam' -r '/project/GCRB/Lee_Lab/shared/CommonGenomes_Indexes/fq/GRCh38/genome.fa' -g '/project/GCRB/Lee_Lab/shared/WilliamV/2025_02_26_N2K8_14450_0/RawFiles/call-transcripts-cufflinks.sh-1.0.0/ov4_BYK_plus_BI_4h_REP2_merge_transcripts/merge-transcripts-cufflinks.sh-1.0.0/merged.gtf'
